src/main/scala/com/amazon/deequ/dqdl/execution/DefaultOperandEvaluator.scala

```diff
@@ -0,0 +1,150 @@
+/**
+ * Copyright 2025 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ * use this file except in compliance with the License. A copy of the License
+ * is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ */
+
+package com.amazon.deequ.dqdl.execution
+
+import com.amazon.deequ.dqdl.execution.ResultType.ResultType
+import org.apache.log4j.Logger
+import software.amazon.glue.dqdl.model.DQRule
+import software.amazon.glue.dqdl.model.condition.number.{AtomicNumberOperand, BinaryExpressionOperand, NumericOperand, OperandEvaluator}
```

Comment:
nit: In a future PR, can we remove block imports? Better still, can we add a configuration to the build process such that it fails the build if there are block imports? These type of imports make it difficult to reorder imports and to apply automatic merge resolution.